run_dir/design/flowcells.html

```diff
@@ -11,7 +11,15 @@
 
 <div class="container-fluid" id="flowcell_list">
   <div id="fc_table_div">
-      <h1><span id="page_title">Flowcells</span> </h1>
+      <h1><span id="page_title">Flowcells</span>
+          {% if all %}
+            <small>Showing all flowcells</small>
+            <a class="btn btn-default" href="/flowcells">Filter to only last 6 months</a>
```

Comment:
Maybe the text could be rephrased? 'Show last 6 months only' sounds a tiny bit better, but its up to you 😄